distribution/kubernetes/vector.yaml

```diff
@@ -133,7 +135,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-
+            
```

Comment:
This is undesired, andy idea how to fix this empty line?

Reply:
Let me look, seems introduced by the makefile

Reply:
Right, this is unrelated, let's merge this and I'll deal with this later.

Reply:
I think I see why actually, I might have fixed it with the other charts I had pulled.